k8s/nodes/prod-compute.json

```diff
@@ -1,10 +1,10 @@
 {
-    "kind": "Node",
-    "apiVersion": "v1",
-    "metadata": {
-        "name": "airqo-k8s-worker-1",
-        "labels": {
-            "node-type": "compute"
-        }
+  "kind": "Node",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "airqo-k8s-worker-0",
+    "labels": {
+      "node-type": "compute"
     }
-}
\ No newline at end of file
+  }
```

Comment:
[update compute and highmem nodes](https://github.com/airqo-platform/AirQo-api/pull/1279/commits/a222f7184c34e481f44fab544b413fd301241ef1)